collada/scene.py

```diff
@@ -327,10 +327,18 @@ def __init__(self, id, children=None, transforms=None, xmlnode=None):
           contain any object that inherits from :class:`collada.scene.Transform`
         :param xmlnode:
           When loaded, the xmlnode it comes from
+        :param name:
+          If given, sets the node name property, if missing (None) then then
+          node name property is set from the id value (if not None)
 
         """
         self.id = id
         """The unique string identifier for the node"""
+        if name is not None: 
+            self.name = name
+        else
+            self.name = id
```

Comment:
self.name = id if name is None else name

Reply:
Ok, sorry - I ran the test against the main branch insted of my new branch...